guides/cppguide.xml

```diff
@@ -3054,68 +3054,66 @@ Thomas Voss
         or:
       </p>
       <CODE_SNIPPET>
-        ReturnType LongClassName::really_really_really_long_function_name(
-            Type par_name1,  // 4 space indent
+        auto ClassName::really_really_really_long_function_name(
+            Type par_name1,
             Type par_name2,
-            Type par_name3)
+            Type par_name3) -> ReturnType
         {
-            do_something();  // 4 space indent
+            do_something();
             ...
         }
       </CODE_SNIPPET>
       <p>
         Some points to note:
       </p>
       <ul>
-        <li> The return type is always on the same line as the
-             function name.
-             </li>
+        <li> <code>void</code> or <code>auto</code>  is always on the same line
+             as the function name.
+        </li>
         <li> The open parenthesis is always on the same line as the
              function name.
-             </li>
+        </li>
         <li> There is never a space between the function name and the
              open parenthesis.
-             </li>
+        </li>
         <li> There is never a space between the parentheses and the
              parameters.
-             </li>
-        <li> The open curly brace is always on the line following the last parameter.
-             </li>
+        </li>
+        <li> The open curly brace is always on the line following the last parameter
+             or return type.
+       </li>
         <li> The close curly brace is either on the last line by itself
              or (if other style rules permit) on the same line as the
              open curly brace.
-             </li>
+       </li>
         <li> All parameters should be named, with identical names in the
              declaration and implementation. (Except where unused parameters
              are suppressed in the implementation.)
-             </li>
+       </li>
         <li> All parameters should be in a single line if possible, otherwise:
             <ul>
-                <li> Wrap groups of parameters into the next line, and align the 
```

Comment:
We're not doing grouping at all anymore?

Reply:
We're aligning by an indent, not with the first parameter

Reply:
I still think this removes the “you may group related parameters” bit?